index.ts

```diff
@@ -146,120 +144,57 @@ export async function exportEventsToBigQuery(events: ProcessedPluginEvent[], { g
                 distinct_id,
                 team_id,
                 uuid,
+                timestamp,
                 elements,
                 ..._discard
             } = event
             const ip = properties?.['$ip'] || event.ip
-            const timestamp = event.timestamp || properties?.timestamp
             let ingestedProperties = properties
 
             const shouldExportElementsForEvent =
                 eventName === '$autocapture' || config.exportElementsOnAnyEvent === 'Yes'
- 
 
+
+            const elementsToExport = shouldExportElementsForEvent ? elements : []
             const object: { json: Record<string, any>; insertId?: string } = {
                 json: {
                     uuid,
                     event: eventName,
                     properties: JSON.stringify(ingestedProperties || {}),
-                    elements: JSON.stringify(shouldExportElementsForEvent && elements ? elements : {}),
+                    elements: JSON.stringify(elementsToExport),
                     set: JSON.stringify($set || {}),
                     set_once: JSON.stringify($set_once || {}),
                     distinct_id,
                     team_id,
                     ip,
+                    site_url: '',
```

Comment:
Context on this change?  Is there value in always uploading a blank column?

Reply:
Fixes a potential issue I introduced in the previous PR. We stopped sending `site_url` with the event onEvent gets but this plugin was still sending it as undefined.

I removed it [here](https://github.com/PostHog/bigquery-plugin/commit/0fb438afcc850dc3f4f7f65285430110123a1381) but decided to keep sending it as an empty string to be safe. Unsure if BQ will complain about a value missing.